react-app/src/App.js

```diff
@@ -1,8 +1,9 @@
 import React, { Component } from 'react';
 import { useState } from "react";
 import './App.css';
+import Title from './Components/Title';
   
-
+// why are we using a class component instead of functional component here?? 
```

Comment:
why are we using a class component instead of functional component here?

Reply:
Both are fine. I prefer functional components for a couple reasons. 
1. I think Functional style is easier to write and understand
2. Classes don't really "exist" in Javascript. JS was created as a functional-first language, rather than a class-first language (read; object-oriented programming OOP).

However many developers learned OOP and use OOP style languages (Java, C#, Ruby, PHP, etc.), and prefer the class-style because of this. Hence why you'll see a bunch of examples and tutorials written in class-based style.

Reply:
I see, thank you